dali/python/nvidia/dali/ops.py

```diff
@@ -428,6 +426,13 @@ class _DaliOperatorMeta(type):
     def __doc__(self):
         return _docstring_generator(self)
 
+def _check_arg_input(schema, op_name, name):
+    if name == "name":
```

Comment:
What is this check?

Reply:
Argument called `"name"` is accepted by all operators and it's passed to a particular instance/call. It's the only non-data-node argument passed to `__call__`.